NetCoreTeamCity/Models/TestOccurrences.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace NetCoreTeamCity.Models
+﻿using System.Collections.Generic;
+
+namespace NetCoreTeamCity.Models
 {
     public class TestOccurrences
```

Comment:
We won't need this class anymore - like other complex lists returned by API, this TestOccurrenceItems from response should be rebinded to simple IList of TestOccurrence. Please see Property model for example